### PR TITLE
Show events map on events list page

### PR DESCRIPTION
**Pull request by maurizi**

### CODE REVIEW

File: src/nyc_trees/apps/event/templates/event/event_list_page.html [hunk 1]
```diff
@@ -39,8 +39,12 @@
                 {% include "event/partials/event_list.html" with event_list=all_events %}
             {% endif %}
           </div>
-          <div class="tab-pane" id="map">
+          <div class="tab-pane" id="map-tab" style="position: relative; width: 100%; height: 440px;">
+            <div id="map" style="width: 100%; height: 400px;">
+            </div>
+            <div id="action-bar" style="position: absolute; width: 100%; height: 40px;"></div>
```
Comment (maurizi) [inline src/nyc_trees/apps/event/templates/event/event_list_page.html]: @jfrankl I'm not a huge fan of inline styling, but we haven't really styled any of our map stuff yet and I didn't want to trip you up.

Should I leave these in here, or pull them out to SCSS?

File: src/nyc_trees/apps/event/templates/event/partials/event_popup.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<a href="{{ event.get_absolute_url }}" class="event block item rightarrow">
+    <div class="row">
+        <div class="event-date col-xs-3">
+            <div class="event-month">{{ event.begins_at.date|date:"M" }}</div>
+            <div class="event-day h4">{{ event.begins_at.date|date:"d" }}</div>
+        </div>
+        <div class="event-details col-xs-6">
+            <h5 class="event-detail event-detail--title">{{ event.title }}</h5>
+            <div class="event-detail">{{ event.group.name }}</div>
```
Comment (maurizi) [inline src/nyc_trees/apps/event/templates/event/partials/event_popup.html]: @jfrankl Is there any other info we need on the action bar here?  (page 93 of the wireframes)

**Comment by maurizi** [comment]
@azavea-bot rebuild

**Comment by RickMohr** [comment]
I think you should compute a bounding box of the events and use `map.fitBounds` so they fit the map (but not closer than `zoom.NEIGHBORHOOD`)

File: src/nyc_trees/apps/event/templates/event/event_list_page.html (hunk 1 above)
Comment (jfrankl) [inline src/nyc_trees/apps/event/templates/event/event_list_page.html]: Either way is fine. I don't mind you adding classes or writing css—I can just make changes down the road as needed.

**Comment by maurizi** [comment]
@RickMohr all of your comments have been addressed.

**Comment by RickMohr** [comment]
+1 Great to see we can zoom to extent purely client-side

**Pull request merged**